.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,297 @@
+            if (!(Test-Path "C:\Program Files (x86)\Microsoft Visual Studio\2017\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin")) {
+              #Write-Host "ERROR***: Missing C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+              Write-Host "ERROR***: Missing C:\Program Files (x86)\Microsoft Visual Studio\2017\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+              exit $LastExitCode
+            }
+            #$env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+            $env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2017\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+          } elseif ($env:TILEDB_GA_IMAGE_NAME -eq "windows-2019") {
+            #if (!(Test-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin")) {
+            if (!(Test-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin")) {
+              #Write-Host "ERROR***: Missing C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+              Write-Host "ERROR***: Missing C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+              exit $LastExitCode
+            }
+            #$env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+            $env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+          } else {
+            Write-Host "Unknown image name: '$($env:TILEDB_GA_IMAGE_NAME)'"
+            exit $LastExitCode
+          }
+          
+          # allow double-checking path
+          cmd /c "echo $PATH"
+
+          # TODO DEBUG move this back in the TILEDB_S3 section
+          # currently we do not run S3 tests on Windows because tests time out (minio+azure slow?)
+          #& "$env:BUILD_SOURCESDIRECTORY\scripts\install-minio.ps1"
+
+          $bootstrapOptions = "-EnableVerbose -EnableStaticTileDB -EnableBuildDeps"
+          if ($env:TILEDB_S3 -eq "ON") {
+            $bootstrapOptions = "-EnableS3 " + $bootstrapOptions
+          }
+          if ($env:TILEDB_AZURE -eq "ON") {
+            $bootstrapOptions = "-EnableAzure " + $bootstrapOptions
+          }
+          if ($env:TILEDB_GCS -eq "ON") {
+            $bootstrapOptions = "-EnableGCS " + $bootstrapOptions
+            #NOTE: GCS simulator not yet actually in place.

```

Comment:
I assume this will be visited in other story right?

Reply:
Yes, there is another story for GCS support on windows.